backend/deploy.js

```diff
@@ -19,15 +20,15 @@ exports.deployPages = function (socket, data) {
 
   var process = spawn("./ghpages_deploy.sh", args);
 
-  output.lineOutput(socket, process, 'deploy-logs', 18);
-  output.lineError(socket, process, 'err-logs');
+  socketHandler.handleLineOutput(socket, process, 'github-deploy-logs', 18);
```

Comment:
They seem to be doing similar things that's why I asked to move it If it was possible. Anyways seems fine.